code/modules/projectiles/ammunition/energy/laser.dm

```diff
@@ -24,6 +24,11 @@
 /obj/item/ammo_casing/energy/laser/hos
 	e_cost = 120
 
+/obj/item/ammo_casing/energy/laser/musket
+	projectile_type = /obj/projectile/beam/laser
+	e_cost = 1000
+	select_name = "kill"
```

Comment:
```suggestion
	e_cost = 1000
```
You don't need to have these because the parent ammo casing object already has the values for `projectile_type` and `select_name` set to what you are currently using.

Reply:
This is liable to change, but thanks for the warning.